site/source/docs/porting/asyncify.rst

```diff
@@ -4,123 +4,109 @@
 Asyncify
 ========================
 
-In general, you want to make your code as asynchronous-friendly as possible, e.g. by never calling sleep, having a single main loop and so on.
+Asyncify lets synchronous C or C++ code do asynchronous Web API calls. For example, you can write ``emscripten_sleep(100)`` and it will return to the main browser event loop for 100ms, during which browser events can be handled, etc. (without returning to the event loop, no events will be dispatched). Asyncify does this by automatically transforming the compiled code into something that can be paused and resumed, and handles pausing and resuming for you.
```

Comment:
Can you line wrap all these paragraphs as 80 columns?

Reply:
Done.